File: output/ChatGPT/codenet/Java/C++/s685537842.cpp
#include <iostream>

int main() {
    int a, b, t;
    std::cin >> a >> b >> t;
    double time = 0;
    int count = 0;
    while (time < t + 0.5) {
        count += b;
        time += a;
        if (time > t + 0.5) {
            count -= b;
            break;
        }
    }
    std::cout << count << std::endl;
    
    return 0;
}

File: output/CodeGen/codenet/Java/Python/s685537842.py
def main():
    a = int(input())
    b = int(input())
    t = int(input())
    time = 0
    count = 0
    while time < t + 0.5:
        count += b
        time += a
        if time > t + 0.5:
            count -= b
            break
    print(count)

main()